### ISSUE TITLE
Exception Thrown when Editing Inline Initialization

### ISSUE THREAD

**Issue by kstatz12** [OmniSharp/omnisharp-roslyn#1928]
Whenever i am initializing an object via inline initializaion ie

```
var obj = new Bar {
};
```
this exception is thrown by the LSP
```
LSP :: Error from the Language Server: Internal Error - System.NullReferenceException: Object reference not set to an instance of an object
  at OmniSharp.Roslyn.CSharp.Services.Signatures.SignatureHelpService.GetInvocation (Microsoft.CodeAnalysis.Document document, OmniSharp.Models.Request request) [0x00290] in <acaf666e7b22409ca6cbf1827f8d2eeb>:0 
  at OmniSharp.Roslyn.CSharp.Services.Signatures.SignatureHelpService.Handle (OmniSharp.Models.SignatureHelp.SignatureHelpRequest request) [0x000b2] in <acaf666e7b22409ca6cbf1827f8d2eeb>:0 
  at OmniSharp.LanguageServerProtocol.Handlers.OmniSharpSignatureHelpHandler.Handle (OmniSharp.Extensions.LanguageServer.Protocol.Models.SignatureHelpParams request, System.Threading.CancellationToken token) [0x000c8] in <3f6b59b22f8c45f9b774c888b79da1bd>:0 
  at OmniSharp.Extensions.LanguageServer.Server.Pipelines.ResolveCommandPipeline`2[TRequest,TResponse].Handle (TRequest request, System.Threading.CancellationToken cancellationToken, MediatR.RequestHandlerDelegate`1[TResponse] next) [0x0006d] in <1238dd34c3f54a5080acfb1001dda78e>:0 
  at MediatR.Pipeline.RequestPreProcessorBehavior`2[TRequest,TResponse].Handle (TRequest request, System.Threading.CancellationToken cancellationToken, MediatR.RequestHandlerDelegate`1[TResponse] next) [0x0013e] in <856ae299ee314304bc3affd17d8d5868>:0 
  at MediatR.Pipeline.RequestPostProcessorBehavior`2[TRequest,TResponse].Handle (TRequest request, System.Threading.CancellationToken cancellationToken, MediatR.RequestHandlerDelegate`1[TResponse] next) [0x0007e] in <856ae299ee314304bc3affd17d8d5868>:0 
  at MediatR.Pipeline.RequestExceptionProcessorBehavior`2[TRequest,TResponse].Handle (TRequest request, System.Threading.CancellationToken cancellationToken, MediatR.RequestHandlerDelegate`1[TResponse] next) [0x00084] in <856ae299ee314304bc3affd17d8d5868>:0 
  at MediatR.Pipeline.RequestExceptionProcessorBehavior`2[TRequest,TResponse].Handle (TRequest request, System.Threading.CancellationToken cancellationToken, MediatR.RequestHandlerDelegate`1[TResponse] next) [0x00276] in <856ae299ee314304bc3affd17d8d5868>:0 
  at MediatR.Pipeline.RequestExceptionActionProcessorBehavior`2[TRequest,TResponse].Handle (TRequest request, System.Threading.CancellationToken cancellationToken, MediatR.RequestHandlerDelegate`1[TResponse] next) [0x00084] in <856ae299ee314304bc3affd17d8d5868>:0 
  at MediatR.Pipeline.RequestExceptionActionProcessorBehavior`2[TRequest,TResponse].Handle (TRequest request, System.Threading.CancellationToken cancellationToken, MediatR.RequestHandlerDelegate`1[TResponse] next) [0x001e7] in <856ae299ee314304bc3affd17d8d5868>:0 
  at OmniSharp.Extensions.JsonRpc.RequestRouterBase`1[TDescriptor].<RouteRequest>g__InnerRoute|5_0 (Microsoft.Extensions.DependencyInjection.IServiceScopeFactory serviceScopeFactory, OmniSharp.Extensions.JsonRpc.Server.Request request, TDescriptor descriptor, System.Object params, System.Threading.CancellationToken token, Microsoft.Extensions.Logging.ILogger logger) [0x000f2] in <fba9e575570149efaee3484292339424>:0 
  at OmniSharp.Extensions.JsonRpc.RequestRouterBase`1[TDescriptor].RouteRequest (OmniSharp.Extensions.JsonRpc.IRequestDescriptor`1[TDescriptor] descriptors, OmniSharp.Extensions.JsonRpc.Server.Request request, System.Threading.CancellationToken token) [0x005f3] in <fba9e575570149efaee3484292339424>:0 
  at OmniSharp.Extensions.JsonRpc.InputHandler+<>c__DisplayClass38_0.<RouteRequest>b__5 (System.Threading.CancellationToken ct) [0x000e8] in <fba9e575570149efaee3484292339424>:0  (Internal Error)
```

Im using Emacs and im on the 1.37.1.beta33 build

**Comment by kstatz12** [comment]
This is resolved in the 1.37.1 release

**Comment by elxbarbosa** [comment]
I can confirm that
